relayer/src/chain/cosmos/types/account.rs

```diff
@@ -0,0 +1,70 @@
+use core::fmt;
+use ibc_proto::cosmos::auth::v1beta1::BaseAccount;
+
+/// Wrapper for account number and sequence number.
+///
+/// More fields may be added later.
+#[derive(Clone, Debug, PartialEq)]
+pub struct Account {
```

Comment:
This file is moved from `chain::cosmos::acccount` to `chain::cosmos::types::acccount`.